lib/src/builders/application.dart

```diff
@@ -39,7 +39,7 @@ class ApplicationUpdateBuilder extends UpdateBuilder<Application> {
   Map<String, Object?> build() => {
         if (customInstallUrl != null) 'custom_install_url': customInstallUrl!.toString(),
         if (description != null) 'description': description,
-        if (roleConnectionsVerificationUrl != null) 'role_connections_verification_url': roleConnectionsVerificationUrl!.toString(),
+        if (roleConnectionsVerificationUrl != null) 'role_connections_verification_url': roleConnectionsVerificationUrl?.toString(),
```

Comment:
This should be left unchanged. We just checked it's not null, and we should error if it is (we don't want to include the field but make it null).

Reply:
It can be null, according to https://github.com/discord/discord-api-docs/pull/6484 / developer portal requests

Reply:
We'll wait for that to be merged (or at least acknowledged by Discord).

Also, if it's meant to be nullable, it should use a sentinel in the constructor and an `if (identical(...))` check.